src/modules/Server/public/Show-SdnVfpPortConfig.ps1

```diff
@@ -0,0 +1,40 @@
+function Show-SdnVfpPortConfig {
+    <#
+    .SYNOPSIS
+        Enumerates the VFP layers, groups and rules contained within Virtual Filtering Platform (VFP) for the specified port.
+    .PARAMETER PortId
+        The Port ID GUID for the network interface.
+    .PARAMETER Direction
+        Specify the direction
+    #>
+
+    [CmdletBinding(DefaultParameterSetName = 'Default')]
+    param (
+        [Parameter(Mandatory = $true, ParameterSetName = 'Default')]
+        [GUID]$PortId,
+
+        [Parameter(Mandatory = $false, ParameterSetName = 'Default')]
```

Comment:
@luyaof thanks for catching this! I have pushed a code fix to address this, and allows defining both **In** or **Out** for values, or if do not define `-Direction`, it will return both in and out groups.